### ISSUE TITLE
fiction.live question

### ISSUE THREAD

**Issue by DragoLord19D** [JimmXinu/FanFicFare#697]
Why does it update all fiction.live stories when the haven't updated in a while?

**Comment by JimmXinu** [comment]
@HazelSh, do you know anything about this?

(I don't know anything about fiction.live.)

**Comment by HazelSh** [comment]
@DragoLord19D, please double-check that you're in "Update if New Chapters" mode. 

I'll look into it anyway, though. I suspect something's going on there.

**Comment by DragoLord19D** [comment]
I am, that was the first thing I checked.

**Comment by HazelSh** [comment]
Had uni stuff being stressful, sorry. But I've reproduced it locally, can confirm there's something funny going on with detecting new chapters.

**Comment by DragoLord19D** [comment]
Cool, it’s not just me messing something up.

**Comment by JimmXinu** [comment]
@HazelSh - Ever figure out anything about this?

**Comment by HazelSh** [comment]
Gotta admit there has been no progress on this

**Comment by JimmXinu** [comment]
Fair enough.  I don't know anything about the site, is this issue closer to a major problem or a minor annoyance?

**Comment by HazelSh** [comment]
I think more of a minor annoyance for now? I do mean to come back to it but so long as stories download I'm not treating it as critical. I have some ideas for improving the fanficfare/fictionlive integration w/r/t chapters getting extended, but I've been pretty severely stressed and near burnout most of this year and haven't had time/energy to work on it further.

**Comment by JimmXinu** [comment]
No problem.  I'll leave this open for future reference.  Take care of yourself.
